src/Microsoft.Data.Sqlite/project.json

```diff
@@ -21,7 +21,7 @@
     ]
   },
   "dependencies": {
-    "SQLite": "3.12.2",
+    "SQLitePCLRaw.bundle_e_sqlite3": "1.1.0-pre20160928150051",
```

Comment:
Or would we need to do [something like this](https://github.com/aspnet/Microsoft.Data.Sqlite/issues/148#issuecomment-223383478)?